src/Kdyby/Translation/Console/CreateTableCommand.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Translation\Console;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Doctrine\DBAL\Types\Type;
+use Kdyby;
+use Nette;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+
+
+/**
+ * @author Filip Procházka <filip@prochazka.su>
+ */
+class CreateTableCommand extends Command
+{
+
+	/** @var Nette\DI\Container */
+	private $serviceLocator;
+
+	/** @var Connection */
+	private $connection;
+
+	/** @var AbstractSchemaManager */
+	private $schemaManager;
+
+	/** @var string */
+	public $table;
+
+	/** @var string */
+	public $key;
+
+	/** @var string */
+	public $locale;
+
+	/** @var string */
+	public $message;
+
+	/** @var string */
+	public $updatedAt;
+
+	protected function configure()
+	{
+		$this->setName('kdyby:translation-create-table')
```

Comment:
Hm, to je divný. @fprochazka proč to není namespace? :stuck_out_tongue: 
Já bych obojí přejmenoval, BC se dá zachovat pomocí setAlias.